habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/util/UvUtil.java

```diff
@@ -0,0 +1,36 @@
+package org.technologybrewery.habushu.util;
+
+import java.io.File;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.logging.Log;
+import org.technologybrewery.habushu.PythonPackageAndDependencyManagerSetup;
+
+/**
+ * Common utility methods for handling TOML uv.
+ */
+public final class UvUtil {
+    /**
+     * Specifies the semver compliant requirement for the version of uv that
+     * must be installed and available for Habushu to use.
+     */
+    public static final String UV_VERSION_REQUIREMENT = ">=0.5.0";
+
+    public static String findCurrentVirtualEnvironmentFullPathForUv(String pythonVersion, String pythonPackageAndDependencyManager,
```

Comment:
A: If you like the rename of the poetry method, would make this match:
```suggestion
    public static String findVirtualEnvironmentAbsolutePath(String pythonVersion, String pythonPackageAndDependencyManager,
```